src/hotspot/share/oops/arrayKlass.hpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1997, 2024, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1997, 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
arrayKlass.hpp isn't changed, is this left over from a previous iteration?

Reply:
yes, it was something that my copyright script thought I changed from merging some previous changes.